lib/knapsack_pro/config/ci/codeship.rb

```diff
@@ -25,6 +25,10 @@ def branch
         def project_dir
           # not provided
         end
+
+        def detected
+          ENV['CI_NAME'] == 'codeship' ? self.class : nil
```

Comment:
This is ok. Should stay as it is. The key `CI_NAME` does not include the CI provider name so we can't assume what is the CI provider. We need to check the value is `codeship`.